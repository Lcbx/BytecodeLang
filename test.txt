16909060
3242
"hello"
great
"\"I love RocknRoll\""
2