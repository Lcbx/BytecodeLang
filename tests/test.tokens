NAME: def
NAME: hello
OP: =
INT: 34
NAME: def
NAME: dear
OP: =
FLOAT: 34.5
NAME: def
NAME: equal
OP: =
NAME: hello
OP: ==
AUX: (
NAME: dear
OP: -
FLOAT: 0.5
AUX: )
EOF: line 2