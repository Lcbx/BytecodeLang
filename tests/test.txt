hi "grosse" + "string"
hi 45 + 5 * 8 < 1000 or 34
hi "hello" + 23
hi 45<54 and (1<1 or 1==1)