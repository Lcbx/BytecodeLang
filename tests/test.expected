compile_Out:
0 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\Lang\compiler\vm_simulator.py", line 22, in <module>
    opName = opcodes[opcode].name
KeyError: 100